<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>hex_convertor_widget</class>
 <widget class="QWidget" name="hex_convertor_widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>494</width>
    <height>85</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>494</width>
    <height>85</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>HEX Convertor</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>471</width>
     <height>61</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="hex_lbl_">
      <property name="text">
       <string>HEX</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="hex_txt_"/>
    </item>
    <item row="0" column="2">
     <widget class="QComboBox" name="type_cb_">
      <property name="editable">
       <bool>false</bool>
      </property>
      <item>
       <property name="text">
        <string>UINT32</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>INT32</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>FLOAT32</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="0" column="3">
     <widget class="QPushButton" name="to_hex_btn_">
      <property name="text">
       <string>To HEX</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="number_lbl_">
      <property name="text">
       <string>Number</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="number_txt_"/>
    </item>
    <item row="1" column="3">
     <widget class="QPushButton" name="to_number_btn_">
      <property name="text">
       <string>To Number</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
